crates/solidity/testing/snapshots/cst_output/TrailingTrivia/only_single_comment/input.sol

```diff
@@ -0,0 +1,2 @@
+ // first
```

Comment:
nit: are the whitespaces intended here? I wonder if we should update the name of the test if so.

Reply:
I wanted to say that only a single comment is collected from the two; I've changed this to `only_until_newline` to better convey the meaning that we should only collect... well, until the newline, as part of the trailing trivia.